server/src/test/java/org/apache/druid/server/QueryResourceTest.java

```diff
@@ -896,7 +896,9 @@ public void testTooManyQuery() throws InterruptedException
     assertAsyncResponseAndCountdownOrBlockForever(
         SIMPLE_TIMESERIES_QUERY,
         waitAllFinished,
-        response -> Assert.assertEquals(Response.Status.OK.getStatusCode(), response.getStatus())
+        response -> {
+          Assert.assertEquals(Response.Status.OK.getStatusCode(), response.getStatus());
+        }
```

Comment:
This looks superfluous?  I'm guessing you added some lines and then didn't clean up the `{}` or is there actually a change in here?

Reply:
oops yea, i started to change something but then didn't end up doing it and forgot to roll back